disputer/test/index.js

```diff
@@ -69,25 +69,15 @@ contract("index.js", function(accounts) {
   });
 
   it("Completes one iteration without throwing an error", async function() {
-    const address = emp.address;
-
-    const priceFeedConfig = defaultPricefeedConfig;
-
     let errorThrown;
     try {
-      await Poll.run(address, false, 10_000, priceFeedConfig);
+      await Poll.run(emp.address, 0, defaultPriceFeedConfig);
       errorThrown = false;
     } catch (err) {
       console.error(err);
       errorThrown = true;
     }
     assert.isFalse(errorThrown);
-  });
-
-  it("Sets token allowances correctly", async function() {
```

Comment:
This test (and the previous one) were joined as there was a lot of overlapping logic. 

Additionally, the need to introduce the `await waitForLogger(Logger);` function call in the index loop break resulted in tests failing if sequentially tested on the main index file. After much testing, this is as a result of the `logger.end()` call in the Logger.js [here](https://github.com/UMAprotocol/protocol/blob/66cfd69e046e175e39ac58a82c10d12e56f3128d/financial-templates-lib/logger/Logger.js#L38) which acts to terminate the node stream from the Winston instance. If this line is called from a testing file the following error was generated:
```
NodeError: write after end
```

The solution is to not close the stream as sequential writes result in this error. I did find a solution to this by modifying the Logger.js implementation to check if the script is running in a testing env. if it was, then the `logger.end()` command can be skipped which solved this error. However, I chose to not implement this change and rather join the tests as this pollutes the Logger JS implementation for this bug.

Reply:
So does this mean we cannot add additional unit tests to this script?

Reply:
right now, ye. we cant add any more to this test script. This is fine for now I think.